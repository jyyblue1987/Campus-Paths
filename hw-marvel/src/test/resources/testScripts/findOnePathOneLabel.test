# checks if the output is correct for paths with single labels

CreateGraph g

AddNode g n1
AddNode g n2
AddNode g n3
AddNode g n4

AddEdge g n1 n2 e12
AddEdge g n2 n3 e23
AddEdge g n3 n4 e34

FindPath g n1 n4