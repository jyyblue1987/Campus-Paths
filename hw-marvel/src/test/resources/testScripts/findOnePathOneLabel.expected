# checks if the output is correct for paths with single labels

created graph g

added node n1 to g
added node n2 to g
added node n3 to g
added node n4 to g

added edge e12 from n1 to n2 in g
added edge e23 from n2 to n3 in g
added edge e34 from n3 to n4 in g

path from n1 to n4:
n1 to n2 via e12
n2 to n3 via e23
n3 to n4 via e34